### DISCUSSION TITLE
Can the value of mode_devi_jobs["press"] be set to a negative value?

### DISCUSSION THREAD

**Discussion by Saberve** [deepmodeling/dpgen#1040] [Q&A]
![image](https://user-images.githubusercontent.com/53855907/203129604-d12baba5-e88b-467f-9118-c360723de59d.png)
Can the value of mode_devi_jobs["press"] be set to a negative value? 
When press is positive it means it is compressing the box, can it be set to a negative value to expand the box, like stretching.

**Comment by AnguseZhang** [comment]
If such setting is physical and practical in LAMMPS, it should work in DP-GEN as well.

**Reply by Saberve** [reply]
thanks!
